docs/en-US/component/cascader.md

```diff
@@ -44,7 +44,36 @@ cascader/last-level
 
 ## Multiple Selection
 
-Add`:props="props"`in tag and set `props : { multiple = true }` to use multiple selection.
+Add`:props="props"`in tag and set data `props : { multiple = true }` to use multiple selection.<br/>
+Example:<br/>
+You have to use it like this:
```

Comment:
Actually let's remove the inline `<br/>` tag, you can insert an empty line after each line to the the desired result.

The title can be simple as I mentioned, `Do` `Don't do` 

line#48 should be removed.